src/frontend/src/catalog.rs

```diff
@@ -121,6 +130,38 @@ impl FrontendCatalogManager {
         let tg_key = tg_key.as_bytes();
 
         self.backend_cache_invalidator.invalidate_key(tg_key).await;
+
+        let key = TableNameKey::new(catalog, schema, table);
+        self.backend_cache_invalidator
+            .invalidate_key(&key.as_raw_key())
+            .await;
+        debug!(
+            "invalidated cache key: {}",
+            String::from_utf8_lossy(&key.as_raw_key())
+        );
+
+        let key = TableInfoKey::new(table_id);
+        self.backend_cache_invalidator
+            .invalidate_key(&key.as_raw_key())
+            .await;
+        debug!(
+            "invalidated cache key: {}",
+            String::from_utf8_lossy(&key.as_raw_key())
+        );
+
+        let key = TableRegionKey::new(table_id);
+        self.backend_cache_invalidator
+            .invalidate_key(&key.as_raw_key())
+            .await;
+        debug!(
+            "invalidated cache key: {}",
+            String::from_utf8_lossy(&key.as_raw_key())
+        );
```

Comment:
Can we atomically invalidate all the keys at once? Otherwise, data inconsistency may occur.

Reply:
We can't, the underlying cache lib we use don't provide such functionality. I don't think it'll create inconsistency issues because the values are all orthogonality.